examples/thoughts/.env

```diff
@@ -1 +1 @@
-DATABASE_URL=postgresql://postgres:devpass@localhost:5432/postgres
+DATABASE_URL=postgresql://postgres:devpass@localhost:5432/thoughts-dev
```

Comment:
I should have prob added to a .gitignore instead, but at least didn't want it to pollute the `postres` db accidentally for anyone else in meantime :)